ansible/roles/mn-evo-services/tasks/main.yml

```diff
@@ -22,13 +22,13 @@
       - name: platform-logs
         definitions:
           - logs:
-              - '{{ mn_evo_services_path }}/logs/*.log'
+              - '{{ mn_evo_services_path }}/logs/*.log*'
```

Comment:
Do we need the star at the end for logrotate?

Reply:
I replaced it while trying to understand why the container wasn't releasing the file after it was renamed so we could still capture logs from files with a date suffix. The last two hosts exhibiting this behaviour (seed nodes) have been fixed now so I guess we can remove it.